substreams-sink-postgres/schemas /eth-block-meta/schema.sql

```diff
@@ -0,0 +1,49 @@
+CREATE SCHEMA eth;
+
+CREATE TABLE eth.block_meta (
```

Comment:
Can we change the name of this table? meta mostly used for basic details of something, here we are saving all details of block

Reply:
block_details?